app/src/main/java/com/example/pigkeeper/Turn Order.kt

```diff
@@ -0,0 +1,140 @@
+package com.example.pigkeeper
+
+import android.content.Intent
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.constraintlayout.widget.ConstraintSet
+import kotlin.math.ceil
+
+class TurnOrderActivity : AppCompatActivity() {
+    var SitOut = false
+
+    //Change this to get the names of the people from New Players Screen
+    var Players = listOf("Jason","Sam","Peter","Luck","Avery","May","Tim","Judith","Garen","Donny")

```

Comment:
Need to pull this from shared data structure which would be populated in New Players Screen